src/settings.js

```diff
@@ -0,0 +1,5 @@
+const defaultSettings = {
+  skull: false
+};
+
+export default defaultSettings;
```

Comment:
if you like to have one file for settings related functions/object, it's fine to call it `settings.js`.
But if you only have one default export, it's better to give it the same name as the file (or rename the file).